qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/common/cropping/cropping_presenter.py

```diff
@@ -53,11 +53,16 @@ def on_combo_changed(self, index):
             self.custom_calfile_enabled = False
             self.custom_spectra_enabled = True
             self.set_custom_widgets_visibility(False, True)
-        else:  # texture
+        elif index ==4:  # texture 20 grouping
```

Comment:
I didn't think there was a Texture20 in the list, just Texture and Texture30. Am I missing something?

Reply:
That branch is for the original texture ROI (20 spectra - 10 from each bank) - I considered renaming the original Texture ROI to Texture20, so maybe I left that comment in, but I still think it's helpful.